continuous_integration/gpuci/axis.yaml

```diff
@@ -2,6 +2,7 @@ PYTHON_VER:
 - "3.9"
 - "3.10"
 - "3.11"
+- "3.12"
```

Comment:
Previously we've had just two python versions running the GPU tests (lowest & highest available versions, currently that's python 3.9 and 3.12). Do we want to remove these intermediate python versions, and keep testing only against two python versions? If so, we'll edit these lines.